### PR TITLE
Feat/saved

### PR DESCRIPTION
**Pull request by willhobson85**
Created functionality to save word tiles and change between views

#Pull Request

- [X] New feature (adds functionality)

#Please describe the tests that you ran to verify your changes.

- [X] I tested my changes in the browser
- [ ] I tested with Cypress

#Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new bugs
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any other comments, questions or concerns?

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -37,19 +42,26 @@ class App extends Component {
     this.setState({wordList: removeTile})
   }
 
-  //save method (need to pass into MainContainer)
+  saveTile = (id) => {
+    const saveCard = this.state.wordList.filter(card => card.id === id).flat()
+    this.setState({savedWords: [...this.state.savedWords, saveCard[0]]})
```
Comment (piperjarrett) [inline src/Components/App/App.js:47]: would a find be better here if you're just looking for one? Just a suggestion but not super necessary

File: src/Components/App/App.js [hunk 2]
```diff
@@ -37,19 +42,26 @@ class App extends Component {
...
+  }
   
   render() {
-    console.log(this.state)
+    console.log('saved words', this.state.savedWords)
```
Comment (piperjarrett) [inline src/Components/App/App.js:51]: reminder to delete console.logs!

File: src/Components/SavedPage/SavedPage.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Tile from "../Tile/Tile.js"
+import './SavedPage.css'
+
+const SavedContainer = ({ wordList, removeTile, saveTile, savedWords }) => {
+
```
Comment (piperjarrett) [inline src/Components/SavedPage/SavedPage.js:6]: can take out wordlist if you are not using it!

**Review by piperjarrett — COMMENTED** [review]
This looks really good Will! Just a few comments on things you could do but aren't super necessary!

File: src/Components/App/App.js (hunk 2 above)
Comment (willhobson85) [reply src/Components/App/App.js:51]: Thank you, I will get that resolved.

File: src/Components/SavedPage/SavedPage.js (hunk 3 above)
Comment (willhobson85) [reply src/Components/SavedPage/SavedPage.js:6]: Thank you! Missed that one!

**Pull request merged**